testing/testData/gen-job.xml

```diff
@@ -68,20 +68,23 @@
                 cron(&quot;*/${EVERY_XMIN} * * * *&quot;)
                 }
 
-                wrappers {
-                if (Math.abs(random.nextInt() % 100) + 1 &lt;= singleP) {
-                configure { node -&gt;
-                node / &apos;buildWrappers&apos; / &apos;org.jenkinsci.plugins.mesos.MesosSingleUseSlave&apos;()
-                }
-                }
+               scm {
+                    git {
+                        remote {
+                        name(&apos;origin&apos;)
+                        url(&apos;https://github.com/mesosphere/marathon.git&apos;)
+                        }
+                        branches(&apos;v1.6.352&apos;)
+                    }
                 }
 
                 steps {
-                shell(&quot;echo &apos;hello, world&apos;; sleep ${sleepDur}&quot;)
-                }
+                   shell(&apos; export SBT_OPTS=&quot;-Xmx750M -Xms750M -XX:+UseConcMarkSweepGC -XX:+CMSClassUnloadingEnabled -Xss2M&quot;; curl -LO https://piccolo.link/sbt-1.1.2.tgz; tar -zxf sbt*.tgz; sbt/bin/sbt compile; &apos;)
```

Comment:
Any concerns with `;` instead of `&&`? The latter will stop early if a previous step failed.

Reply:
element tree was having issues parsing with && so i replaced it with ;